src/app/jobs/components/job-list/job-list.component.html

```diff
@@ -16,47 +16,62 @@
       <p>Sorry, we were not able to retrieve your jobs. Please try again by refreshing the page.</p>
     </div> -->
+    <table mat-table [dataSource]="dataSource" matSort class="table">
 
-    <table mat-table [dataSource]="jobs | async">
-      <ng-container matColumnDef="id">
-        <th mat-header-cell *matHeaderCellDef>#</th>
-        <td mat-cell *matCellDef="let job">{{job.id}}</td>
-      </ng-container>
-
-      <ng-container matColumnDef="type">
-        <th mat-header-cell *matHeaderCellDef>Type</th>
-        <td mat-cell *matCellDef="let job">{{job.type}}</td>
+      <ng-container matColumnDef="product_name">
+        <th mat-header-cell *matHeaderCellDef mat-sort-header>Product</th>
+        <td mat-cell *matCellDef="let job">{{job.product_name}}</td>
       </ng-container>
 
-      <ng-container matColumnDef="product">
-        <th mat-header-cell *matHeaderCellDef>Product</th>
-        <td mat-cell *matCellDef="let job">{{job.product_name}}</td>
+      <ng-container matColumnDef="status">
+        <th mat-header-cell *matHeaderCellDef mat-sort-header>State</th>
+        <td mat-cell *matCellDef="let job">
+          <div class="status-item" [class.status-item-pending]="job?.status === 'PENDING' || job?.status === 'STARTED'">
+            {{job.status | titlecase}}
+          </div>
+          <mat-progress-spinner
+            *ngIf="job?.status === 'PENDING' || job?.status === 'STARTED'"
+            mode="indeterminate"
+            diameter="25"
+            class="status-item">
+          </mat-progress-spinner>
+        </td>
       </ng-container>
 
-      <ng-container matColumnDef="state">
-        <th mat-header-cell *matHeaderCellDef>State</th>
-        <td mat-cell *matCellDef="let job">{{job.status | titlecase}}</td>
+      <ng-container matColumnDef="created">
+        <th mat-header-cell *matHeaderCellDef mat-sort-header>Started</th>
+        <td mat-cell *matCellDef="let job">{{job.created | date : 'd LLL yyyy, HH:mm'}}</td>
       </ng-container>
 
       <ng-container matColumnDef="details">
-        <th mat-header-cell *matHeaderCellDef>Details</th>
+        <th mat-header-cell *matHeaderCellDef></th>
         <td mat-cell *matCellDef="let job">
-          <button mat-button color="primary" [routerLink]="['/jobs', job.id]">
+          <button
+            mat-button
+            *ngIf="(sessionState$ | async).authenticated; else disabledDetails"
+            color="primary"
+            [routerLink]="['/jobs', job.id]">
             Details
           </button>
+          <ng-template #disabledDetails>
+            <div matTooltip="Please log in or register to use this functionality!">
+              <button mat-button disabled>
+                Details
+              </button>
+            </div>
+          </ng-template>
         </td>
       </ng-container>
 
-      <tr mat-header-row *matHeaderRowDef="displayedColumns"></tr>
+      <tr mat-header-row *matHeaderRowDef="displayedColumns sticky: true"></tr>
```

Comment:
Cool, but if you change the header of the table to sticky, maybe we should do it in all the tables, to have a common style :)